scripts/devops_tasks/test_run_samples.py

```diff
@@ -100,6 +100,18 @@
         "mgmt_topic_async.py",
         "proxy_async.py",
         "receive_deferred_message_queue_async.py"
+    ],
+    "azure-communication-chat": [
+        "chat_client_sample_async.py",
+        "chat_client_sample.py",
+        "chat_thread_client_sample_async.py",
+        "chat_thread_client_sample.py"
+    ],
+    "azure-communication-phonenumbers": [
```

Comment:
Why are we ignoring these phone number samples?

Reply:
Samples actually run, so we want to avoid running the purchase and release actions just as we do on the tests